packages/instant/public/index.html

```diff
@@ -58,6 +63,7 @@
                 </button>
             </div>
         </div>
+        <img id="augurScreenshot" class="full-screen" src="./augur_screenshot.png" />
```

Comment:
I think this is good for testing, but I'm hesitant to actually commit this.  Your call though though

Reply:
kk I can remove